components/Title.js

```diff
@@ -0,0 +1,7 @@
+export default function Title({ children }) {
+  return (
+    <h3 className="title">
+      {children}
+    </h3>
+  );
+}
```

Comment:
What is the criteria to define this subtitle component as a h3? Does it make sense based on the hierarchy?

Reply:
This is based on the style guide page in Figma, there are styles for each typography